src/main/java/org/openstreetmap/atlas/vectortiles/TippecanoeExporter.java

```diff
@@ -0,0 +1,148 @@
+package org.openstreetmap.atlas.vectortiles;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.commons.exec.CommandLine;
+import org.apache.commons.exec.DefaultExecutor;
+import org.apache.commons.exec.PumpStreamHandler;
+import org.apache.commons.exec.util.StringUtils;
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.openstreetmap.atlas.geography.atlas.geojson.LineDelimitedGeoJsonConverter;
+import org.openstreetmap.atlas.utilities.runtime.CommandMap;
+import org.openstreetmap.atlas.utilities.time.Time;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This CLI will take a directory full of atlases and export it into an MBTiles file full of Mapbox
+ * vector tiles. It creates intermediary line-delimited GeoJSON, and then it ultimately drives
+ * tippecanoe to do the vector tile creation.
+ *
+ * @author hallahan
+ */
+public final class TippecanoeExporter extends LineDelimitedGeoJsonConverter
```

Comment:
Same comment here as for the Super class. How can we have this be split into two, one that runs the actual logic, and can save to a WritableResource, and another one that runs it command-style?

Reply:
As we chatted in person, you can actually already save a given atlas as ld geojson to a WritableResource, which this CLI calls. the predicate and mutator in the settings class is now exposed as public, that way you can set this same thing up in Spark.